common/models/service-instance.js

```diff
@@ -141,4 +141,311 @@ module.exports = function(ServiceInstance) {
       description: 'List npm modules'
     }
   );
+
+  ServiceInstance.remoteMethod(
+    'agentTraces',
+    {
+      http: {path: '/processes/:pk/agentTraces', verb: 'get'},
+      isStatic: false,
+      accepts: [
+        {
+          arg: 'pk',
```

Comment:
this is a sub-route of instance i.e. `/instances/:id/processes/:pk/agentTraces` The `:id` key is already being used so I'm using `:pk` instead.